packages/core/src/components/tabs/tabs.tsx

```diff
@@ -141,15 +158,70 @@ export class Tabs extends AbstractPureComponent<ITabsProps, ITabsState> {
                     role="tablist"
                 >
                     {tabIndicator}
-                    {tabTitles}
+                    {this.props.overflow ? (
+                        <OverflowList
+                            // @ts-ignore
+                            items={tabTitles}
+                            collapseFrom={"start"}
+                            observeParents={true}
```

Comment:
prefer prop defaults. do not change this; let the user override with `overflowListProps` _if needed_.

Reply:
(see #2505 for context)